src/components/ui/Tracks.module.css

```
@@ -0,0 +1,337 @@
+
+.bars span:nth-child(2) {
+  height: 70%;
+  animation-delay: -0.35s;
+}
+
+.bars span:nth-child(3) {
+  height: 55%;
+  animation-delay: -0.65s;
+}
+
+@keyframes tracksBar {
```

Stylelint (17.14.0)

[error] Expected keyframe name "tracksBar" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)